tests/unit/test_flatten_fhir_example.py

```diff
@@ -203,6 +203,7 @@ def flatten_scalars(self: DomainResource) -> dict:
 
 def flatten_references(self: DomainResource) -> dict:
     """Convert the DomainResource instance to a dictionary."""
+
```

Comment:
add medication administration test? Maybe ask for dummy data or use a version of our current data with dummy keys/values

Reply:
The data-framer is very much a piece of hacky scripting that makes a bunch of assumptions and inevitably breaks when seeing new code. 

I see no reason to write 'dataframer' specific tests since the best test is giving it a new piece of data and watching it throw an exception.